web/src/test/java/com/graphhopper/http/resources/CustomWeightingRouteResourceLMTest.java

```diff
@@ -97,7 +97,8 @@ public void testCustomWeighting() {
                 "profile: car_custom\n" +
                 "priority:\n" +
                 "  road_class:\n" +
-                "    secondary: 2\n";
+                "    secondary: 1\n" +
+                "    '*': 0.5\n";
```

Comment:
We can now live without being able to "prefer" secondary roads explicitly ... but this is now slightly more work.

Reply:
Yes its a bit more verbose now, but also for me everything is a lot more clear now.